packages/webdriver/src/utils.js

```diff
@@ -302,7 +302,7 @@ export function environmentDetector ({ hostname, capabilities, requestedCapabili
  */
 export function getErrorFromResponseBody (body) {
     if (!body) {
-        return null
+        return new Error('unknown error')
```

Comment:
It might be better to give clearer error message, also we know what happens, for example `Response has empty body`

Reply:
I agree.